pkg/util/fs_util.go

```diff
@@ -534,14 +534,13 @@ func AddVolumePathToWhitelist(path string) {
 // 	1. If <src> is a remote file URL:
 // 		- destination will have permissions of 0600
 // 		- If remote file has HTTP Last-Modified header, we set the mtime of the file to that timestamp
-func DownloadFileToDest(rawurl, dest string) error {
+func DownloadFileToDest(rawurl, dest string, uid, gid int64) error {
```

Comment:
Does it make sense to have int64 here (and in `GetUserGroup`), when there are `uint32` everywhere else?

Reply:
in `copy.go` the *ExecuteCommand* method passes int64 uid and gid params to the util.CopyDir method, which passes this around to many other places such as `fs_utils.go`...
I'm not sure what should be the actual datatype (it probably doesn't matter), but making it consistent across the code-base would require touching many functions, and I am not sure if it's a good idea. WDYT?